### PR TITLE
Blog Post on First "Carpentries Chat" in NZ

### PR DESCRIPTION
**Pull request by megan-guidry**
Here is a summary of the Carpentries Chat we had in Dunedin back on August 30th.  This is the first time I am trying to submit a blog post via Git so let me know if I need to change something.

### CODE REVIEW

File: _posts/2019-09-20_First_NZ_Carpentries_Chat_Blog_Post.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
+
+---
+layout: page
+authors: ["Megan Guidry", "Murray Cadzow", "Antje Lubcke"]
+teaser: "New Zealand holds its first 'Carpentries Chat' in Dunedin"
+title: "New Zealand holds its first 'Carpentries Chat' in Dunedin"
+date: 2019-09-20
```
Comment (kariljordan) [inline _posts/2019-09-20_First_NZ_Carpentries_Chat_Blog_Post.md]: @megan-guidry excellent post! Can you change the date in the file name to 2019-09-24, and change it in the blog header as well? We will merge this post on Tuesday, 24-September.

File: _posts/2019-09-20_First_NZ_Carpentries_Chat_Blog_Post.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
+
+---
+layout: page
+authors: ["Megan Guidry", "Murray Cadzow", "Antje Lubcke"]
+teaser: "New Zealand holds its first 'Carpentries Chat' in Dunedin"
+title: "New Zealand holds its first 'Carpentries Chat' in Dunedin"
```
Comment (kariljordan) [inline _posts/2019-09-20_First_NZ_Carpentries_Chat_Blog_Post.md]: May I recommend that the teaser be different from the title of the blog post? What's one thing you're excited about that others can be excited about too when reading this post? Make that the teaser:)

File: _posts/2019-09-20_First_NZ_Carpentries_Chat_Blog_Post.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+date: 2019-09-20
+time: "09:00:00"
+tags: ["Community", "Discussion", "Carpentries"]
```
Comment (kariljordan) [inline _posts/2019-09-20_First_NZ_Carpentries_Chat_Blog_Post.md]: May I recommend adding "New Zealand" as a tag for this post?

**Comment by serahkiburu** [comment]
@megan-guidry, thank you so much for coordinating the penning of this post and congratulations on a fantastic first Carpentries Chat, and on being a first time contributor on our website repo! 🚀  Woohoo Megan! 

I'll merge the post first thing tomorrow morning. Thanks again 🙌

**Pull request merged**